cmake/resumable_tasks.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CheckSymbolExists)
+
+if (CMAKE_SYSTEM_NAME MATCHES "^Linux")
+    check_symbol_exists("getcontext" "ucontext.h" HAVE_UCONTEXT)
+    if (HAVE_UCONTEXT)
```

Comment:
I would rename `HAVE_UCONTEXT` to `_tbb_have_ucontext` since it is not meant to be used globally.